esmvalcore/preprocessor/_regrid_esmpy.py

```diff
@@ -161,6 +161,11 @@ class ESMPyAreaWeighted(_ESMPyScheme):
 
     Does not support lazy regridding.
 
+    .. deprecated:: 2.12.0
```

Comment:
Would it make sense to also print a warning if these schemes are used in a recipe? E.g., add an `ESMValCoreDeprecationWarning` to the `_ESMPyRegridder` code?

Reply:
Done in 286f43d